examples/simple/main.tf

```
@@ -8,6 +8,13 @@ data "aws_ami" "ecs_ami" {
   }
 }
 
+resource "aws_security_group" "test_group" {
+  name        = "test_group"
```

This is fine for now, but we could run into problems later if we have multiple tests running and both try to create a security group named `test_group`. `var.test_name` is a generated name from terratest, so should avoid any potential conflicts. 
```suggestion
  name        = var.test_name
```